src/components/nav-user/resources.ts

```diff
@@ -0,0 +1,5 @@
+export const CSS = {
+  textContainer: "text-container",
+  userFullName: "user-full-name",
```

Comment:
seems weird with `userFullName` and `userUsername`. Can we make these consistent? Maybe just `username` and `fullname`?